Source/WebCore/Modules/webtransport/WebTransportBidirectionalStreamConstructionParameters.h

```diff
@@ -41,6 +41,7 @@ struct WebTransportBidirectionalStreamConstructionParameters {
     WEBCORE_EXPORT WebTransportBidirectionalStreamConstructionParameters(WebTransportBidirectionalStreamConstructionParameters&&);
     WEBCORE_EXPORT WebTransportBidirectionalStreamConstructionParameters& operator=(WebTransportBidirectionalStreamConstructionParameters&&);
     WEBCORE_EXPORT ~WebTransportBidirectionalStreamConstructionParameters();
+    WebTransportBidirectionalStreamConstructionParameters isolatedCopy() && { return { identifier, Ref { sink } }; }
```

Comment:
`&&` doesn't seem needed / helpful?

Reply:
The struct only has move constructor, cannot construct result with `WebTransportBidirectionalStreamConstructionParameters { identifier,  sink }`